src/User32.Desktop/User32+WM_MESSAGES.cs

```diff
@@ -0,0 +1,982 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the Windows Messages constants.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Windows Messages
+        /// Defined in winuser.h from Windows SDK v6.1
+        /// Documentation pulled from MSDN.
+        /// </summary>
+        public enum WM : int
+        {
+            /// <summary>
+            /// The WM_NULL message performs no operation. An application sends the WM_NULL message if it wants to post a message that the recipient window will ignore.
+            /// </summary>
+            NULL = 0x0000,
```

Comment:
Per [naming guidelines](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#naming) these enum values should be named exactly as their native constant equivalents. We shouldn't split WM_NULL into a WM type and a NULL member name.
I'm on the fence about the `WM` type name itself. It seems like WindowMessage might be preferable.